Makefile

```diff
@@ -277,7 +278,7 @@ coverage-run-js:
 	$(RM) -r out/$(BUILDTYPE)/.coverage
 	$(MAKE) coverage-build-js
 	-NODE_V8_COVERAGE=out/$(BUILDTYPE)/.coverage CI_SKIP_TESTS=$(COV_SKIP_TESTS) \
-	  $(MAKE) jstest
+	        TEST_CI_ARGS="$(TEST_CI_ARGS) --type=coverage" $(MAKE) jstest
```

Comment:
I think it would probably make sense to also specify:

`FLAKY_TESTS="dontcare"`

Reply:
k will add.

Reply:
based on @refack's comment I'll leave out setting dontcare completely for now.